### DISCUSSION TITLE
Welcome to iqoption Discussions!

### DISCUSSION THREAD

**Discussion by Muhammad-1990** [Muhammad-1990/iqoption#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Muhammad-1990** [comment]
Hi,

yes it would have to be a series... check out the zigzagSupRes script for a quick array/series, sorry for the lack of comments but il try and clean up that script...

**Reply by atlasoser** [reply]
hello, i have a little question, how to get total number of bars per certain period or each condition by using bars_since(), for example number of bars when ma 7 crossup ma 14 and crossdown, its already can be plotted by get_value, but i confuse how to obtain each total number of each previous condition

**Comment by Agorapronto1212** [comment]
Hello!

**Comment by nanopicker** [comment]
hi, i have a idea
but I am struggling to build simple script for IQ Option, Here my logic two horizontal lines on chart that represent the high and low of the previous completed 5-minute candle, no matter what timeframe your chart is currently on.
Retrieves the high and low of the last fully completed 5-minute candle.
Deletes the old lines from the chart.
Draws two new horizontal lines:

CAN U HELP ME

**Comment by Agorapronto1212** [comment]
Hello, how are you? Send the code.

Obter o Outlook para Android<https://aka.ms/AAb9ysg>
________________________________
